src/components/tiles/TilesList.tsx

```diff
@@ -99,6 +100,7 @@ export function TilesList({ navigation, styles: instanceStyles }: Props) {
         />
       )}
       <View style={styles.tilesView}>
+        {messagesTileEnabled && <MessagesTile navigation={navigation} />}
```

Comment:
Will need to support multiple message tiles from the `appConfig.homeTab.messageTiles` array.

Also maybe consider renaming `MessagesTile` to `MessageTile` to align with multiple `messageTiles` in the appConfig or the other way around, should we rename it in appConfig?  Or maybe not worry about it at all...  😁

Reply:
+1 on prepping for their possibly being multiple Message(s)Tiles.

Also: I think (as a default) this should go down by the `circleTiles` instead of up at the top.

Reply:
Moved down to circles tile and added support for multiple tiles but might need to make some edits depending on my assumptions of the API.